bundles/org.openhab.core.thing/src/main/resources/OH-INF/i18n/SystemProfiles_ru.properties

```diff
@@ -5,9 +5,9 @@ profile.config.system.offset.offset.label = Смещение
 profile.config.system.offset.offset.description = Смещение (просто число или с ед. измерения), применяемое к состоянию по отношению к элементу. Отрицательное смещение будет применено в обратном направлении.
 profile-type.system.hysteresis.label = Гистерезис
 profile.config.system.hysteresis.lower.label = Нижняя граница
-profile.config.system.hysteresis.lower.description = Приводит к состоянию OFF, если значение меньше нижней границы (просто число или с ед. измерения).
+profile.config.system.hysteresis.lower.description = Приводит к состоянию OFF, если значение меньше или равно нижней границе (просто число или с ед. измерения).
```

Comment:
Please remove these translations (only keep the XML changes and the English) and add it later using crowdin. Otherwise it'll not show up there.